2025.02-dev/fpm/entrypoint.sh

```diff
@@ -39,8 +39,16 @@ file_env() {
     unset "$fileVar"
 }
 
+# initialize the email configuration
 sh /setup_msmtp.sh
 
+# ensure we have a logfile writeable by www-data
+FRIENDICA_LOGFILE=${FRIENDICA_LOGFILE:-/var/www/html/friendica.log}
+if sudo -u www-data test -w "$(dirname "$FRIENDICA_LOGFILE")"; then
+   touch "$FRIENDICA_LOGFILE"
+   chown www-data:www-data "$FRIENDICA_LOGFILE"
+fi
```

Comment:
Two suggestions/options:
If it's just supposed to try to create it and not do anything except continue if it fails, one could skip the conditional and needing to install `sudo` and just try to create it:
```
touch "$FRIENDICA_LOGFILE" || true
chown --silent www-data:www-data "$FRIENDICA_LOGFILE" || true
```
(`--silent` should only be there if we want to suppress error messages)

Alternately, instead of using `sudo` I'm guessing `su` or `runuser` are available out of the box. I'd suggest the latter if possible since it's deescalating privileges.